modules/setting-management/src/Volo.Abp.SettingManagement.Application.Contracts/Volo/Abp/SettingManagement/SettingManagementPermissionDefinitionProvider.cs

```diff
@@ -10,12 +12,24 @@ public class SettingManagementPermissionDefinitionProvider : PermissionDefinitio
         public override void Define(IPermissionDefinitionContext context)
         {
             var moduleGroup = context.AddGroup(SettingManagementPermissions.GroupName, L("Permission:SettingManagement"));
-            moduleGroup.AddPermission(SettingManagementPermissions.Emailing, L("Permission:Emailing"), multiTenancySide: MultiTenancySides.Host);
+            var emailingPermission = moduleGroup.AddPermission(SettingManagementPermissions.Emailing, L("Permission:Emailing"));
+
+            if (IsTenantAvailable(context))
```

Comment:
Permission definition providers runs only once. So,  you can not conditionally add permissions like that.